bin/install/install-dcos-cli.sh

```diff
@@ -66,12 +66,29 @@ check_pip_version()
     fi
 }
 
+check_dcoscli_version()
+{
+    if [ ! -z "$DCOS_CLI_VERSION" ]; then
+        # result is the larger of the two versions
+        COSMOS_VERSION="0.4.0"
+        # convert the str to numbers, sort, and return the larger
+        result=$(echo -e "$COSMOS_VERSION\n$DCOS_CLI_VERSION" | sed '/^$/d' | sort -nr | head -1)
```

Comment:
the sed regex just removes extra newlines in case `$DCOS_CLI_VERSION` has any.
```
$ echo -e "0.4.0\n1.3.0\n\n0.2.1\n\n" | sed '/^$/d'
0.4.0
1.3.0
0.2.1
```

then `sort -n` "compare according to string numerical value" and `-r` is to "reverse the result of comparisons", so the largest value will be top of the list, and then you can just pop off the head

```
$ echo -e "0.4.0\n1.3.0\n\n0.2.1\n\n" | sed '/^$/d' | sort -nr
1.3.0
0.4.0
0.2.1
```

```
$ echo -e "0.4.0\n1.3.0\n\n0.2.1\n\n" | sed '/^$/d' | sort -nr | head -1
1.3.0
```